cmake/Modules/FindViennaCL.cmake

```diff
@@ -0,0 +1,41 @@
+# - Try to find ViennaCL
+# Once done this will define
+#
+#  VIENNACL_FOUND        - system has ViennaCL
+#  VIENNACL_INCLUDES     - the ViennaCL include directories
+#  OPenCL Variables in addition
+#
+# Redistribution and use is allowed according to the terms of the BSD license.
+# For details see the accompanying COPYING-CMAKE-SCRIPTS file.
```

Comment:
Same holds as for the other file: Copied? Then comment on it. Missing COPYING-CMAKE-SCRIPTS